src/lib/openapi/spec/feedback-schema.ts

```diff
@@ -0,0 +1,27 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const feedbackSchema = {
+    $id: '#/components/schemas/feedbackSchema',
+    type: 'object',
+    additionalProperties: false,
+    required: [],
```

Comment:
no required fields for the feedback?

Reply:
As far as I can tell, pretty much! `feedbackId` is required by one of the endpoints that use this schema, but decided to check that field on its own instead of making two schemas.